### ISSUE TITLE
4.0.4 - Button no Longer opens a Note window. 

### ISSUE THREAD

**Issue by pmruzicka** [xnotepp/xnote#116]
Button no longer functions at all since I updated from 4.0.3 to 4.0.4. So I can no longer add a note. Old notes only appear at the top of the email message with no way to open note in a window. I see no setting that changed from what I had previous - and no new settings that might alter the behavior. So basically, it went from functioning to no longer working with the update. Using Windows 11 22H2. Thunderbird v. 115.4.1

**Comment by opto** [comment]
yes, 115.4.1 did that. I have  working version which I am testing right now

**Comment by opto** [comment]
please try this  https://github.com/xnotepp/xnote/releases/tag/v4.1.3

**Comment by opto** [comment]
there is a new timing problem. The xpi works after installation until the next restart.

Upon restart, TB does not read the main code file and it fails.

**Comment by pmruzicka** [comment]
It won't install. Says it's not verified...

**Comment by opto** [comment]
please try this:
https://github.com/xnotepp/xnote/releases/tag/v4.1.4

the tag says 4.1.4 but I uploaded 4.1.5

extra-addons-cog symbol-load from file.

which OS do you have? I never got a notice about verification. I only get the one about the full permissions, which we alway had.

**Comment by Chealer** [comment]
@pmruzicka: Does this persist with 4.3.38? If so, please provide a screenshot.
